### PR TITLE
basic meteor doc

### PR DESCRIPTION
**Pull request by whiteotter**
Trimmed down version of Node.js doc, messaging adjusted for meteor service.

### CODE REVIEW

**Comment by mlocher** [comment]
I would also add a reference to the Meteor deployment command, and the wrapper script we have available at https://github.com/codeship/scripts/blob/master/deployments/meteor.sh

File: _basic/languages-frameworks/meteor.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+title: Using Meteor In CI/CD with Codeship Basic
+menus:
+  basic/languages:
+    title: Meteor
+    weight: 2
```
Comment (ethangj) [inline _basic/languages-frameworks/meteor.md]: Can you adjust the weight? Would put it after Node/PHP/Rails/Python at least.

File: _basic/languages-frameworks/meteor.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+* include a table of contents
+{:toc}
+
+## Setup
+
+Meteor's default installer requires sudo on Linux. We use a script to change install location and make sudo unnecessary:
+
+```shell
+\curl -sSL https://raw.githubusercontent.com/codeship/scripts/master/packages/meteor.sh | bash -s
+```
```
Comment (ethangj) [inline _basic/languages-frameworks/meteor.md]: Is there any version info we can add here? What version, how to change?

File: _basic/languages-frameworks/meteor.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+### Yarn
+
+You can also use [Yarn](https://yarnpkg.com/en) to install your dependencies as an alternative to npm. Yarn is pre-installed on the build VMs. We configure `yarn` to write into `$HOME/cache/yarn`, which is also cached.
+
+```shell
+meteor npm install -g yarn
+meteor yarn
+```
+
```
Comment (ethangj) [inline _basic/languages-frameworks/meteor.md]: Under Notes And Known issues (see other Basic languages) -- can we add the deployment commands Marko is referencing in his comment as an h4?

**Comment by ethangj** [comment]
I will also reorg to match other PR before merging.

**Comment by whiteotter** [comment]
ok @mlocher (or @ethangj) and for the meteor deployment wrapper, is there anything I should be mentioning for why the wrapper script ought to be used instead of the basic `meteor deploy URL` ?

File: _basic/languages-frameworks/meteor.md (hunk 3 above)
Comment (whiteotter) [inline _basic/languages-frameworks/meteor.md]: made it a full h2 section, lmk if there's any issues with that

File: _basic/languages-frameworks/meteor.md (hunk 2 above)
Comment (whiteotter) [inline _basic/languages-frameworks/meteor.md]: added discussion of `release` flag for utilizing previous versions of the `meteor` tool

File: _basic/languages-frameworks/meteor.md (hunk 1 above)
Comment (whiteotter) [inline _basic/languages-frameworks/meteor.md]: 👍

**Comment by ethangj** [comment]
@whiteotter maybe just mention both as options unless @mlocher has more specific info?

**Pull request merged**